grails-app/migrations/install/changelog-insert-data.xml

```diff
@@ -372,6 +372,46 @@
             <column name="name" value="Partially Received"/>
             <column name="sort_order" valueNumeric="3"/>
         </insert>
+        <insert tableName="event_type">
+            <column name="id" value="abb5b3d2-da3e-4a4c-9e3c-de139147f822"/>
```

Comment:
Are these the same exact IDs as the ones created in develop?

Reply:
I used here exactly what is in the changelog adding this event type